File: Java/1464_Maximum_Product_of_Two_Elements_in_an_Array.java
class Solution {
    public int maxProduct(int[] nums) {
        int v1 = Integer.MIN_VALUE;
        int v2 = Integer.MIN_VALUE;
        for(int i = 0; i < nums.length; i++){
            if(nums[i] > v1){
                v2 = v1;
                v1 = nums[i];
            }
            else if(nums[i] > v2){
                v2 = nums[i];
            }
        }
        return (v1-1)*(v2-1);
    }
}

File: CS/1464_Maximum_Product_of_Two_Elements_in_an_Array.cs
public class Solution {
    public int MaxProduct(int[] nums) {
        int num1 = Int32.MinValue;
        int num2 = Int32.MinValue;
        for(int i = 0; i < nums.Length; i++){
            if(nums[i] >= num1){
                num2 = num1;
                num1 = nums[i];
            }
            else if(nums[i] > num2){
                num2 = nums[i];
            }
        }
        return (num1-1) * (num2-1);
    }
}
